src/OctoshiftCLI.IntegrationTests/AdoCsvToGithub.cs

```diff
@@ -0,0 +1,79 @@
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OctoshiftCLI.IntegrationTests
+{
+    [Collection("Integration Tests")]
+    public class AdoCsvToGithub : AdoToGithub
+    {
+        public AdoCsvToGithub(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task With_Inventory_Report_Csv()
```

Comment:
Couldn't this have been just another method?

Reply:
not sure what you mean? it is another method

Reply:
if you're asking about why I split it up into 2 separate classes (and then introduced a base class to avoid repeating the test setup), it's because "slicing" the tests in the build is easier to do at the class boundary (at least the way it's currently implemented).